Things.md

```diff
@@ -0,0 +1,181 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing. The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.  
+  
+### Table of Contents
+1. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+2. [GroupHoldr](#groupholdr)
+    1. [Basic Architecture](#basic-architecture)
+        1. [Important APIs](#important-apis)
+        2. [Constructor Arguments](#constructor-arguments)
+    3. [Sample Usage](#sample-usage)
+
+# Manipulating Things
```

Comment:
You should mention what the coordinate system is based off of here. People don't inherently know our origin & scaling.